climatepolicyradar/navigator-db-client
# Adds init_data to 0028 migration and retires init_data

This is a breaking change!

- Removes `init_data` as a way to do data migrations
- Adds the functionality of `init_data` to the migration `0028` (not yet released)
- Removes adding a super-user - as testing adds its own and is no longer needed now we have a long running database
- Manual testing success :star: 
   - Works on a clean db
   - Using the dump `staging_2024_02_19_start.sql` and the script `~/work/navigator-scripts/psql/copy-all-data.sql` I was able to see the differences after migration 0028 and 0029 were applied and everything was as expected.
